docs/acquisition.rst

```diff
@@ -0,0 +1,160 @@
+Acquiring multi-echo data
+=========================
+
+Available multi-echo fMRI sequences
+-----------------------------------
+
+Siemens
+```````
+**For Siemens** users, there are two options for Works In Progress (WIPs) Sequences.
+
+* | The Center for Magnetic Resonance Research at the University of Minnesota
+  | provides a custom MR sequence that allows users to collect multiple echoes
+  | (termed **Contrasts**). The sequence and documentation can be `found here`_.
+  | For details on obtaining a license follow `this link`_.
+  | By default the number of contrasts is 1, yielding a single-echo sequence.
+  | In order to collect multiple echoes, increase number of Contrasts on the
+  | **Sequence Tab, Part 1** on the MR console.
+* | The Martinos Center at Harvard also has a MR sequence available, with the
+  | details `available here`_. The number of echoes can be specified on the
+  | **Sequence, Special** tab in this sequence.
+
+.. _found here: https://www.cmrr.umn.edu/multiband/
+.. _this link: http://license.umn.edu/technologies/cmrr_center-for-magnetic-resonance-research-software-for-siemens-mri-scanners
+.. _available here: https://www.nmr.mgh.harvard.edu/software/c2p/sms
+
+GE
+``
+**For GE users**, there are currently two sharable pulse sequences:
+
+* Multi-echo EPI (MEPI) – Software releases: DV24, MP24 and DV25 (with offline recon)
+* | Hyperband Multi-echo EPI (HyperMEPI) - Software releases: DV26, MP26, DV27, RX27
+  | (here hyperband can be deactivated to do simple Multi-echo EPI – online recon)
+
+Please reach out to the GE Research Operation team or each pulse sequence’s
+author to begin the process of obtaining this software.
+More information can be found on the `GE Collaboration Portal`_
+
+Once logged in, go to Groups > GE Works-in-Progress you can find the description
+of the current ATSM (i.e. prototypes).
+
+.. _GE Collaboration Portal: https://collaborate.mr.gehealthcare.com
+
+Other available multi-echo MRI sequences
```

Comment:
This section is a good addition, but missing a few key points.
1. "While tedana allows the input of a T2* map from any source, and a more accurate T2* map should lead to better results, this hasn't been systematically evaluated yet."
2. There are many ways to calculate quantitative T2* maps. You're listing one way and making some claims that aren't obvious to a reader (i.e. get phase data). This doesn't really tell people where to look for more info. I suggest find your favorite reference for this type of T2* map and reference it here.

Reply:
I'm adding in your first point verbatim, but am a little unsure about what to say for the second. In this section I am basically parroting responses to #464, since I don't have actual experience with quantitative T2* mapping. I can include the paper linked in #464 but I'm not really familiar with any others. My own attempts at searching for T2*-mapping papers have mostly come across methods for scanning knees.

Reply:
In that case, here's my suggested re-write of that paragraph. As a general message, I think we should focus on the fact that multi-echo is used in other ways and that some of these ways can calculate potentially useful T2* maps. The original text got specific in ways that left gaps of omission. I've tried to get the main point across without over-specification. (If you like my text, make sure the links are appropriate & work... I'm just drafting here)

"In addition to ME-fMRI, other MR sequences benefit from acquiring multiple
 echoes, including T1-weighted imaging (MEMPRAGE) and susceptibility weighted imaging.
 While most of these kinds of sequences fall outside the purview of this documentation, quantitative T2* mapping is relevant since a baseline T2* map is used in several processing steps including :ref:`optimal combination`.  While the T2* map estimated directly from fMRI time series is noisy, no current study quantifies the benefit to optimal combination or tedana denoising if a higher quality T2* map is used. Some benefit is likely, so, if a T2* map is independently calculated, it can be used as an input to many functions in the tedana workflow.

There are many ways to calculate T2* maps, with some using multi-echo acquisitions. We are not presenting an expansive review of this literature here, but https://doi.org/10.1016/j.ejro.2018.12.006 and https://doi.org/10.1016/j.neuroimage.2012.01.053 are good places to start learning more about this topic."

Reply:
Looks great. Thank you!

Reply:
One really minor suggestion to take or leave. When you reference "Cohen-Adad et al. (2012) and Ruuth et al. (2019)" add links to the DOIs for those papers.

Reply:
There are supposed to be links. I must have messed up the hyperlinking, but I'll fix it now.

Reply:
Ah, I added the actual references below but forgot to actually format the links in the text.